_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kubeflow 1.10 Release Announcement"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.10-release/
+author: "Kubeflow 1.10 Release Team, Dimitris Poulopoulos"
+---
+
+Kubeflow 1.10.0 delivers essential updates that enhance the flexibility, efficiency, and scalability of machine learning
+workflows. The new features span across several components, improving both user experience and system performance.
+
+## Benefits to the ecosystem
+
+* Migration to GitHub Container Registry (ghcr) as default container registry for (most) images
+
+## Highlight features
+
+* Trainer 2.0
+* Model Registry UI
+* Spark Operator as a core Kubeflow component
+* Kubernetes and container security (CISO compatibility)
+* LLM hyper parameter tuning
+
```

Comment:
This post is long, we need to cut some.   I would propose starting with line 28 i.e. cut lines 12-27.   You might remove "upcoming"